Sources/Decoder.swift

```diff
@@ -157,6 +157,9 @@ public struct Decoder {
                 
                 for subJSON in jsonArray {
                     guard let model = T(json: subJSON) else {
+                        if T.self is ArrayPartiallyDecodable.Type {
```

Comment:
I was thinking about that as well, but the problem is that if you apply that logic to primitive types you would not be able to distinguish when you want that to happen, when you create your own type (struct, class, enum) you can specify _"hey I want just this model to be able to be partially parsed"_, but to apply that to primitive types like URL, Dates, Bool etc.. you would have to adopt the protocol to the type via an extension making the type partially decodable throughout the entire code base so you can't have control over it... 

For this case if you need that functionality then you will have to create a wrapper on top of the primitive value like:

```
``` swift
import Gloss

struct PartiallyDecodableBool: Glossy, ArrayPartiallyDecodable {

    let value: Bool
    
    // MARK: - Deserialization
    // ...

    // MARK: - Serialization
}
```